guides/documentations/configuration.md

```diff
@@ -0,0 +1,91 @@
+# How to set your plugin's configuration
+
+The DefaultConfiguration class is used to set your plugin's configuration.
+
+In this part, you will see how you can:
+- Tag your plugin
+- Define the type for your plugin diagram
+- Define a default file name and file extension
+- Define a restrictive folder
+- Set the configuration for the code editor
+
+## Tag your plugin
+
+To help users understand the usage of your plugin, you can tag it using two types of tags:
+- `language`: This tag indicates the programming language used in your plugin.
+- `category`: This tag assigns an appropriate category to your plugin, such as CI, CD, Workflow, and more.
```

Comment:
CI/CD

Reply:
done. Also, should I delete lines 127-192 inside `README.md` at the root, since it's the same explanation as in the `guides/documentations/configuration.md` ?

Reply:
you can remove explanation in Readme and put a link to this doc :)